_posts/2019-04-15-submitting-prs.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Submit Great Pull Requests"
+date: 2019-04-15 15:00:00
+categories: ["agile", "learning", "pull-requests"]
+author: "emily"
+---
+
+Pull Requests let developers tell other team members about changes they've made to a project repository. Once a pull request is created, team members can review the set of changes, discuss potential modifications and even push follow-up commits before the changes are merged into the repository. Therefore, it is important to make sure that your pull requests are easily understandable to the reviewers.
+
+<!--more-->
+
+Fellow developers need to be able to understand what the pull request is trying to achieve, what approach is being taken, and how all of the changed files relate to each other. **In short, a great pull request will:**
+
+- Be short, have a clear title and description, and do only one thing
+- Be reviewed quickly
+- Reduce the probability of introduction a bug into the stable branch
+- Facilitate the on-boarding of new developers
+- Not block other developers
+- Speed up the code review process and consequently, product development
+- Target the right branch, usually either master or develop, depending on the project
+
+In order to make a **great pull request**, follow these Ombu Labs approved tips:
+
+### 1. Make smaller pull requests
+
+Make smaller pull requests. This is the best way to speed up your review time. **The smaller the pull request, the  easier it is for the reviewers to understand** it and provide feedback.
+
+### 2. Break down large pull requests into smaller ones
+
+A study demonstrated that **a good pull request should not have more than 250 lines of changed code**. If the functionality you are working on is too big or complex, it could be helpful to split it into smaller and more manageable pull requests.
```

Comment:
@ercohen14 I think you missed the link then you mention `A study` -> https://opensource.com/article/18/6/anatomy-perfect-pull-request